scripts/validate_solutions.py

```diff
@@ -1,12 +1,18 @@
 #!/usr/bin/env python3
 import importlib.util
 import json
+import os
 import pathlib
 from types import ModuleType
 from typing import Dict, List
 
 import pytest
 
+try:
+    import github
+except ImportError:
+    pass
```

Comment:
```suggestion
    github = None
```
This allows easy to understand `if github:` logic.

Reply:
I will be using `requests` to directly make the API call as there is only one.